### DISCUSSION TITLE
2-0069 skyline has panicked

### DISCUSSION THREAD

**Discussion by NoahMizzet** [Raytwo/ARCropolis#332] [Issues]
![20220504_200925](https://user-images.githubusercontent.com/104953257/166853513-6f12897b-fcc3-4a22-8632-ea32e35dea99.jpg)
I don't know what is going on with skyline but it said to take a screenshot and ask the dev.

**Comment by Raytwo** [comment]
The menu system had trouble writing files to your SD it seems. Make sure to have a SD formatted in FAT32, delete the « html-document » directory next to the romfs directory for Smash in sd:/contents/atmosphere/
